wp-lazy-loading.php

```diff
@@ -134,33 +133,17 @@ function wp_lazy_loading_enabled( $tag_name, $context ) {
  * @return string Converted content with 'loading' attributes added to images.
  */
 function wp_add_lazy_load_attributes( $content, $context = null ) {
-	$tags = array();
-
-	if ( null === $context ) {
-		$context = current_filter();
-	}
-
-	if ( wp_lazy_loading_enabled( 'img', $context ) ) {
-		$tags[] = 'img';
-	}
-
```

Comment:
Thinking we should keep this. `wp_lazy_loading_enabled()` requires context in order to be "most useful". Passing `current_filter()` is quite helpful. For example it would allows adding of the attribute in post_content, but not adding it in comments, etc. This also would let plugins customize it further if/when they call `wp_add_lazy_load_attributes()` directly.